subsys/task_wdt/task_wdt.c

```diff
@@ -65,7 +65,7 @@ static void schedule_next_timeout(int64_t current_ticks)
 #endif
 
 	/* find minimum timeout of all channels */
-	for (int id = 0; id < ARRAY_SIZE(channels); id++) {
+	for (size_t id = 0; id < ARRAY_SIZE(channels); id++) {
```

Comment:
Is this change actually needed (it does not seem to be related to some build warning)? If so, there is another `for` loop below that also uses an `int` iterator, so it would be good to keep both consistent.